setup.py

```diff
@@ -23,11 +23,15 @@
 
 from setuptools import find_packages
 
+
+REQUIREMENTS = open('requirements.txt').read().splitlines()
+
+# Hack for read the docs (TODO : explain why this is needed)
 on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 if on_rtd:
-    shapely_dep = "Shapely<1.5.13"
-else:
-    shapely_dep = "Shapely>=1.5.17,<1.6.dev0"
+    for i, line in enumerate(REQUIREMENTS):
+        if "Shapely>=1.5.17,<1.6.dev0" in line:
```

Comment:
Why this special case?

Reply:
This was a fix introduced because readthedocs cannot build the Shapely -> GEOS libs.
It's not required anymore because we introduced the dedicated requirements_docs.txt for RTD.

Reply:
https://github.com/GeoNode/geonode/pull/3767

Reply:
Thanks, I removed it